docs/using-radicle/pushing-changes.md

```diff
@@ -0,0 +1,46 @@
+---
+id: pushing-changes
+title: Pushing changes
+---
+
+To push code to Radicle, you first need to make sure your shell is configured
+correctly by following the instructions in the [Getting Started][gs] section to
+add the Radicle remote helper.
+
+After the Radicle remote helper is configured, pushing to Radicle is as simple
+as pushing to any other remote, with the Radicle network being known as 'rad'.
+After changes are added and commited in git, they can be pushed to the Radicle
+remote via `git push rad`. This will push your upstream tracking branch. 
+
+Radicle projects also support [branches][br]. Specific branches can be pushed to
```

Comment:
The flow here seems off. I think it should follow a similar flow to the fetch and push guide. So like:
* Ensure they've created a project and link to the section that describes that
* Navigate to working copy
* `git branch` to show branches
* `git remote` to show remotes
* Talk about `rad` pointing to monorepo
* Make a change and commit to a file
* `git push rad`
* Le Fin

Reply:
Added an attempt at this